plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+				'attributes' => [
+					'title'    => __( 'When out of stock', 'woocommerce' ),
+					'property' => 'backorders',
+					'options'  => [
+						[
+							'label' => __( 'Allow purchases', 'woocommerce' ),
+							'value' => 'yes',
+						],
+						[
+							'label' => __(
+								'Allow purchases, but notify customers',
+								'woocommerce'
+							),
+							'value' => 'notify',
+						],
+						[
+							'label' => __( "Don't allow purchases", 'woocommerce' ),
+							'value' => 'no',
+						],
+					],
+				],
+			]
+		);
+		$product_out_of_stock_conditional->add_block(
+			[
+				'id'        => 'product-inventory-email',
+				'blockName' => 'woocommerce/product-inventory-email-field',
+			]
+		);
+
+		$product_inventory_advanced_wrapper->add_block(
+			[
+				'id'         => 'product-limit-purchase',
+				'blockName'  => 'woocommerce/product-checkbox-field',
+				'attributes' => [
+					'title'    => __(
+						'Restrictions',
+						'woocommerce'
+					),
+					'label'    => __(
+						'Limit purchases to 1 item per order',
+						'woocommerce'
+					),
+					'property' => 'sold_individually',
+					'tooltip'  => __(
+						'When checked, customers will be able to purchase only 1 item in a single order. This is particularly useful for items that have limited quantity, like art or handmade goods.',
+						'woocommerce'
+					),
+				],
+			]
+		);
+	}
+
+	/**
+	 * Adds the shipping group blocks to the template.
+	 */
+	private function add_shipping_group_blocks() {
+		$shipping_group = $this->add_group(
+			[
+				'id'         => 'shipping',
+				'order'      => 40,
+				'attributes' => [
+					'id'    => 'shipping',
+					'title' => __( 'Shipping', 'woocommerce' ),
+				],
+			]
+		);
+		$shipping_group->add_block(
+			[
+				'id'         => 'product_variation_notice_shipping_tab',
+				'blockName'  => 'woocommerce/product-has-variations-notice',
+				'attributes' => [
+					'id'         => 'wc-product-notice-has-options',
+					'content'    => __( 'This product has options, such as size or color. You can now manage each variation\'s price and other details individually.', 'woocommerce' ),
+					'buttonText' => __( 'Go to Variations', 'woocommerce' ),
+					'type'       => 'info',
+				],
+			]
+		);
+		// Product Pricing Section.
+		$product_fee_and_dimensions_section = $shipping_group->add_section(
+			[
+				'id'         => 'product-fee-and-dimensions-section',
+				'attributes' => [
+					'title'       => __( 'Fees & dimensions', 'woocommerce' ),
+					'description' => sprintf(
+					/* translators: %1$s: How to get started? link opening tag. %2$s: How to get started? link closing tag.*/
+						__( 'Set up shipping costs and enter dimensions used for accurate rate calculations. %1$sHow to get started?%2$s.', 'woocommerce' ),
+						'<a href="https://woocommerce.com/posts/how-to-calculate-shipping-costs-for-your-woocommerce-store/" target="_blank" rel="noreferrer">',
+						'</a>'
+					),
```

Comment:
Any reason why `sprintf` was used instead of `createInterpolateElement`?

Reply:
Similar reasons to the above `sprintf` comment